### PR TITLE
mount docker container logs directory

### PR DESCRIPTION
**Pull request by tt810**
### What does this PR do?
Add docker container mount directory when running daemonset in kubes.

### Motivation
Logs are not visible to agent with current daemonset configuration.

### Preview link
https://docs-staging.datadoghq.com/ting.tu/update-daemonset-kubes/agent/kubernetes/daemonset_setup/?tab=k8sfile

### CODE REVIEW

File: content/en/agent/kubernetes/daemonset_setup.md [hunk 1]
```diff
@@ -361,7 +377,7 @@ The workaround in this case is to add `hostNetwork: true` in your Agent pod spec
 [8]: /agent/autodiscovery
 [9]: /integrations/amazon_ec2/#configuration
 [10]: /logs
-[11]: /tracing/setup
-[12]: /graphing/infrastructure/process/?tab=kubernetes#installation
-[13]: /agent/kubernetes/dogstatsd
-[14]: https://docs.datadoghq.com/agent/autodiscovery/?tab=kubernetes#setting-up-check-templates
+[11]: https://docs.datadoghq.com/agent/autodiscovery/?tab=kubernetes#setting-up-check-templates
```
Comment (ruthnaebeck) [inline content/en/agent/kubernetes/daemonset_setup.md]:
```suggestion
[11]: /agent/autodiscovery/?tab=kubernetes#setting-up-check-templates
```

**Comment by l0k0ms** [comment]
This will be out with Agent 6.12

**Comment by NBParis** [comment]
LGTM.

@l0k0ms do you know if there are other places (default daemonset, helm charts) where this update should be propagated?

**Comment by l0k0ms** [comment]
@NBParis I would say the in app documentation (in the log section, the Agent section, and the onboarding) the public helm chart doc: https://github.com/helm/charts/tree/master/stable/datadog#enabling-log-collection

**Pull request merged**